### PR TITLE
Added preStartCommand options in mission control charts

### PR DESCRIPTION
**Pull request by ypan887**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Added preStartCommand options in mission control charts. User can now run scripts to import trusted certificates to java keystore from mount point.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/mission-control/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # JFrog Mission-Control Chart Changelog
 All changes to this chart will be documented in this file.
 
+
+## [1.1.9] - Nov 15, 2019
+Support missionControl.preStartCommand for running command before entrypoint starts
```
Comment (eldada) [inline stable/mission-control/CHANGELOG.md]: Add a leading `*`

File: stable/mission-control/values.yaml [hunk 2]
```diff
@@ -334,6 +334,8 @@ missionControl:
   internalPort: 8080
   externalPort: 80
 
+  ## Extra pre-start command to import trusted keystore, need to mount the certificates first
```
Comment (eldada) [inline stable/mission-control/values.yaml]: Use a more general description. The certificate is one example you can show.
Leave the line
```
# preStartCommand:
```
Without a value. Move this example to a comment above (with leading `##`)

**Comment by ypan887** [comment]
HI, Eldad, I have fixed it.

**Comment by eldada** [comment]
@ypan887 - resolve conflicts.

**Pull request merged**